nixos/release-combined.nix

```diff
@@ -106,8 +116,8 @@ in rec {
         (all nixos.tests.xfce)
 
         nixpkgs.tarball
-        (all nixpkgs.emacs)
-        (all nixpkgs.jdk)
+        (all allSupportedNixpkgs.emacs)
+        (all allSupportedNixpkgs.jdk)
```

Comment:
This looks like the wrong place to build these packages. Shouldn't we rather move this to nixpkgs instead of having it in NixOS?